### DISCUSSION TITLE
Install with Homebrew returns with "No available formula with the name "fluidsynth""

### DISCUSSION THREAD

**Discussion by Ranger6012** [FluidSynth/fluidsynth#1075] [Q&A]
I am trying to install fluidsynth using the Homebrew command provided here (brew install fluidsynth). However, using it returns with this

```
Warning: No available formula with the name "fluidsynth".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps
```

How do I fix this and install fluidsynth, Homebrew is installed correctly. I am not sure if this is the case for everything as I only intended to install fluidsynth.

**Comment by derselbst** [comment]
For some reason, Homebrew calls it `fluid-synth`: https://formulae.brew.sh/formula/fluid-synth#default

**Reply by Ranger6012** [reply]
Trying `brew install fluid-synth` returns with the same response of no available formula
